test/e2e/tests/test_process/test_container.py

```diff
@@ -0,0 +1,58 @@
+import pytest
+from sinspqa import sinsp
+from sinspqa.sinsp import assert_events
+from sinspqa.docker import get_container_id
+
+sinsp_filters = ["-f", "evt.category=process and not container.id=host"]
+
+containers = [
+    {
+        'sinsp': sinsp_container,
+        'nginx': {
+            'image': 'nginx:1.14-alpine',
+        }
+    } for sinsp_container in sinsp.generate_specs(args=sinsp_filters)
+]
+
+ids = [ sinsp.generate_id(c['sinsp']) for c in containers ]
+
+@pytest.mark.parametrize("run_containers", containers, indirect=True, ids=ids)
+def test_exec_in_container(run_containers: dict):
+    nginx_container = run_containers['nginx']
+    sinsp_container = run_containers['sinsp']
+
+    container_id = get_container_id(nginx_container)
+
+    nginx_container.exec_run("sleep 5")
+    nginx_container.exec_run("sh -c ls")
+
+    expected_events = [
+        {
+            'container.id': container_id,
+            'evt.args': 'filename=/usr/sbin/nginx ',
+            'evt.category': 'process',
+            'evt.type': 'execve',
+            'proc.exe': 'runc',
+            'proc.cmdline': 'runc:[1:CHILD] init',
```

Comment:
Does this depend on the "outside" system using versions of `runc` that have these types of input arguments and commandlines? I understand it is unlikely that this would change, but do you think it would make sense to try and catch the "final" execve (if it exists) that runs the actual nginx binary instead?

Reply:
Just to double check, that final execve event would the one right below this event, right? So we would only need to remove the first one from the test.

I'd assume the event actually depends on the system the container is running since the event comes from the container itself, but I might be wrong.